WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift

```diff
@@ -10,12 +10,17 @@ final class QuickStartPromptViewController: UIViewController {
     @IBOutlet private weak var siteDescriptionLabel: UILabel!
 
     /// Prompt info
-    @IBOutlet private weak var promptTitleLabel: UILabel!
-    @IBOutlet private weak var promptDescriptionLabel: UILabel!
+    @IBOutlet private(set) weak var promptTitleLabel: UILabel!
+    @IBOutlet private(set) weak var promptDescriptionLabel: UILabel!
 
     /// Buttons
-    @IBOutlet private weak var showMeAroundButton: FancyButton!
-    @IBOutlet private weak var noThanksButton: FancyButton!
+    @IBOutlet private(set) weak var showMeAroundButton: FancyButton!
+    @IBOutlet private(set) weak var noThanksButton: FancyButton!
+
+    /// Constraints
+    @IBOutlet private(set) weak var scrollViewTopVerticalConstraint: NSLayoutConstraint!
+    @IBOutlet weak var scrollViewLeadingConstraint: NSLayoutConstraint!
```

Comment:
I think these two constraints can be private, as they don't appear to be used elsewhere.

Reply:
Done in 2852603ae6866de5d605e74fca4c0463850c4039